controllers/goflowkube/goflowkube_objects.go

```diff
@@ -24,25 +23,53 @@ const configPath = "/etc/goflow-kube"
 const configFile = "config.yaml"
 
 const (
-	healthServiceName       = "health"
-	healthTimeoutSeconds    = 5
-	livenessPeriodSeconds   = 10
-	startupFailureThreshold = 5
-	startupPeriodSeconds    = 10
+	healthServiceName = "health"
+
+/* 	healthTimeoutSeconds    = 5
+livenessPeriodSeconds   = 10
```

Comment:
I added https://github.com/netobserv/flowlogs2metrics/issues/86 to close this gap ... once this is completed we will be able to return those checks.